README.md

```diff
@@ -23,6 +24,229 @@ Terra Toolkit is a utility module used to facilitate independent development of
+  browser.url('/inaccessible-contrast.html');
+  const rules = {
+    'color-contrast': { enabled: false },
+  };
+  expect(browser.axe({ viewports, rules })).to.be.accessible();
+});
+
+it('runs only specified tags', () => {
+  browser.url('/inaccessible-text.html');
+  const runOnly = {
+    type: 'tag',
+    values: ['color-contrast'],
+  };
+  expect(browser.axe({ viewports, runOnly })).to.be.accessible();
+});
+
+it('runs only specified context', () => {
+  browser.url('/inaccessible-contrast.html');
+  let context = 'h1';
+  expect(browser.axe({ viewports, context })).to.not.be.accessible();
+
+  context = 'h2';
+  expect(browser.axe({ viewports, context })).to.be.accessible();
+});
+```
+
+### Visual Regression Tests
+Terra toolkit provides configuration for a visual regression utility. See [documentation](http://webdriver.io/guide/services/visual-regression.html) for more information. Terra Toolkit provides useful wrappers to make setup easier.
+
+Visual regression tests can be written using the commands provided by the [visual regression service](http://webdriver.io/guide/services/visual-regression.html). Example:
+
+```js
+/* global browser, describe, it, expect, viewport */
+describe('button test', () => {
+  // Only test tiny/huge viewports
+  const viewports = viewport('tiny', 'huge');
+  it('checks visual comparison', () => {
+    browser.url('/buttons.html');
+
+    // Verify the button element matches
+    let screenshots = browser.checkElement('button', { viewports });
+    expect(screenshots).to.matchReference();
+
+    // Verify the whole viewport matches
+    let screenshots = browser.checkViewprot({ viewports });
+    expect(screenshots).to.matchReference();
+  });
+});
+```
+
+### Test Helpers
+
+The terra toolkit provides custom assertions and globals to make testing easier.
+
+#### Viewports
+
+The `viewports()` function provides a convenience method for fetching an array of viewports to use to resize the browser or even pass directly to the accessibility and visual regression commands.
+
+##### Usage
+wdio-axe-service enhances an WebdriverIO instance with the following command:
+
+`viewports(name);`
+
+The following options are available:
+
+* **name**:
+  Takes the viewport keys and returns an array of `{ height, width}` objects representing the respective viewport size(s). If no key is passed, returns all viewports.
+
+
+##### Example
+
+```js
+/* global browser, describe, it, expect, viewport */
+describe('Resizing browser', () => {
+  // Only test tiny/huge viewports
+  const viewports = viewport('tiny', 'huge');
+  it('switches viewport sizes', () => {
+    browser.url('/foo.html');
+    viewports.forEach(size, () => {
+      // Resize browser to each viewport size
+      browser.setViewportSize(size);
+    });
+  });
+});
+```
+
+#### Assertions
+2 custom assertions are provided to make validating the output of the visual regression and accesibility commands easier.
```

Comment:
Update '2' to 'Two'

Reply:
Also, will these assertions work if a user changes the wdio.framework from mocha? Might be good to indicate if not.

Reply:
its not so much mocha but rather the chai framework which i configure by default.